zap/src/test/java/org/zaproxy/zap/control/AddOnUnitTest.java

```diff
@@ -60,88 +60,36 @@
     private static final File ZAP_VERSIONS_XML =
             getResourcePath("ZapVersions-deps.xml", AddOnUnitTest.class).toFile();
 
-    @Test
-    @SuppressWarnings("deprecation")
-    void testIsAddon() throws Exception {
-        assertTrue(AddOn.isAddOn("test-alpha-1.zap"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testNotAddonNoState() throws Exception {
-        assertFalse(AddOn.isAddOn("test-1.zap"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testNotAddonBadExt() throws Exception {
-        assertFalse(AddOn.isAddOn("test-beta-1.zip"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testNotAddonBadStatus() throws Exception {
-        assertFalse(AddOn.isAddOn("test-xxx-1.zap"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testNotAddonBadVersion() throws Exception {
-        assertFalse(AddOn.isAddOn("test-beta-A.zap"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testId() throws Exception {
-        AddOn addOn = new AddOn("test-alpha-1.zap");
-        assertThat(addOn.getId(), is("test"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testStatus() throws Exception {
-        AddOn addOn = new AddOn("test-alpha-1.zap");
-        assertThat(addOn.getStatus().name(), is("alpha"));
-    }
-
-    @Test
-    @SuppressWarnings("deprecation")
-    void testVersion() throws Exception {
-        AddOn addOn = new AddOn("test-alpha-1.zap");
-        assertThat(addOn.getVersion().toString(), is(equalTo("1.0.0")));
-        assertThat(addOn.getFileVersion(), is(1));
-    }
-
     @Test
     @SuppressWarnings("deprecation")
```

Comment:
These should be removed now.

Reply:
I'm not sure I'm following up with you sir, what should be removed?

Reply:
The `SuppressWarnings` annotations, the methods should no longer have any deprecation usage.

Reply:
okkk, now I get it... thanks